datadriven/build/jsgpp/datadriven.i

```diff
@@ -39,6 +39,8 @@
 %include "datadriven/src/sgpp/datadriven/scalapack/DataMatrixDistributed.hpp"
 %include "datadriven/src/sgpp/datadriven/scalapack/DataVectorDistributed.hpp"
 
+%ignore *::operator();
```

Comment:
Doesn't this also ignore the `()` operator in other modules (all *.i files get included in one file `jsgpp.i`)? There are some operators which we want to keep. Can you replace the wildcard with the explicit names?